src/queries/SUE/areaService.tsx

```diff
@@ -17,7 +19,21 @@ export const areaService = async () => {
     }
   };
 
-  return await (
-    await fetch(`${serverUrl}/${id}?selectAttributes=postalCodes`, areaServiceFetchObj)
+  const getRS = await (
+    await fetch(`${serverUrl}/${id}?selectAttributes=rs`, areaServiceFetchObj)
+  ).json();
+
+  const postalCodes = await (
+    await fetch(
+      `${serverUrl}/getByRs?rs=${getRS?.rs}*&selectAttributes=postalCodes`,
```

Comment:
what about

```
const rs = (await (
    await fetch(`${serverUrl}/${id}?selectAttributes=rs`, areaServiceFetchObj)
  ).json())?.rs;

...
`${serverUrl}/getByRs?rs=${rs}
...
```

should work, right?

Reply:
I tried this first but it didn't work so I had to change the code to `getRS?.rs`. 🤷‍♂️

Reply:
```
const rs = (await (await fetch(`${serverUrl}/${id}?selectAttributes=rs`, areaServiceFetchObj)).json()).rs;
```

Reply:
Okay, interesting. Maybe I did something wrong, but it's working now. Thank you 👍